future/ansible-conjur-identity-example/0_start.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -ex
+
+function finish {
+  echo 'Removing docker environment'
+  echo '---'
+  docker-compose down -v
+}
+trap finish EXIT
+
+function main {
+  docker-compose up -d conjur-master
+  docker-compose logs -f conjur-master
+}
+
+
+function bar {
+  echo 'bar'
+}
```

Comment:
Do you think we need this function to just print 'bar'?  Would it make more sense to return the string 'bar'?

Reply:
The function this just needs to see 'bar' printed to the screen.